cli/tests/cmd.rs

```diff
@@ -711,15 +711,16 @@ forgetest_ignore!(can_compile_local_spells, |_: TestProject, mut cmd: TestComman
     let dss_exec_lib = "src/DssSpell.sol:DssExecLib:0xfD88CeE74f7D78697775aBDAE53f9Da1559728E4";
 
     cmd.args([
-        "build",
+        "test",
```

Comment:
Is this supposed to build or test? If build, change this back, otherwise change test name

Reply:
used this to manually test against spells mainnet to check if storage caching works.
had to update to "test" because this is an invalid build command otherwise